smoke-test/nodenext/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodenext-smoke-test",
+  "version": "0.0.0",
+  "type": "module",
+  "module": "./dist/smoke-test.js",
+  "dependencies": {
+    "@apollo/server": "../node_modules/@apollo/server",
```

Comment:
The rest of the smoketests install a built tarball — is that OK? or is this reading from inside smoke-test anyway i guess... huh.

Reply:
Simplified the reproduction a bit, and you're right that it's just looking up.
[cee95bb](https://github.com/apollographql/apollo-server/pull/6731/commits/cee95bbcc0034feb0253357aa3319923f2f5b0e6)